craco.config.js

```diff
@@ -1,19 +1,29 @@
+const { addBeforeLoader, loaderByName } = require('@craco/craco');
+
 module.exports = {
   webpack: {
     configure: (webpackConfig) => {
-      // support for wasm
-      webpackConfig.module.rules.push({
-        test: /\.wasm$/,
-        type: 'webassembly/sync',
+      const wasmExtensionRegExp = /\.wasm$/;
```

Comment:
this can't be correct, we're using a different webpack configuration now, please check `develop` for the correct config